tests/trunc-double-uniform.ispc

```diff
@@ -0,0 +1,127 @@
+
+export void f_f(uniform float RET[], uniform float aFOO[]) {
+
```

Comment:
We don't typically use vertical white space in he beginning of the function.

Reply:
Removed.